chef/cookbooks/hyperv/recipes/config_ceilometer.rb

```diff
@@ -63,7 +25,7 @@
   end
 end
 
-template "#{node[:openstack][:config].gsub(/\\/, "/")}/ceilometer.conf" do
+template "#{node[:openstack][:ceilometer][:config].gsub(/\\/, "/")}/ceilometer.conf" do
```

Comment:
Performance/StringReplacement: Use `tr` instead of `gsub`. (https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code)